File: hammer-interaction/src/main/java/dev/vini2003/hammer/interaction/HI.java
package dev.vini2003.hammer.interaction;

import dev.vini2003.hammer.interaction.registry.common.HIEvents;
import dev.vini2003.hammer.interaction.registry.common.HINetworking;
import net.fabricmc.api.ModInitializer;

public class HI implements ModInitializer {
	@Override
	public void onInitialize() {
		HIEvents.init();
		HINetworking.init();
	}
}

File: hammer-interaction/src/main/kotlin/dev/vini2003/hammer/interaction/HI.kt
package dev.vini2003.hammer.interaction


import dev.vini2003.hammer.interaction.registry.common.HICommands
import dev.vini2003.hammer.interaction.registry.common.HIEvents
import dev.vini2003.hammer.interaction.registry.common.HINetworking
import net.fabricmc.api.ModInitializer

object HI : ModInitializer {
	override fun onInitialize() {
		HICommands.init()
		HIEvents.init()
		HINetworking.init()
	}
}